src/gui/qgslayermetadataresultsproxymodel.h

```diff
@@ -63,6 +73,7 @@ class GUI_EXPORT QgsLayerMetadataResultsProxyModel : public QSortFilterProxyMode
 
     QgsRectangle mFilterExtent;
     QString mFilterString;
+    QString mFilterGeometryTypeName;
```

Comment:
I think using a string here is messy -- it'd be better to use the geometry type enum (with a seperate bool for rasters, or use a map layer type filter for those instead)

Reply:
done!